### PR TITLE
#203 Update color theme & logo

### LINKED ISSUE #203: Update color theme & logo
Goal: update the Uploader branding to match the February arbimon.org launch

Suggested changes:
- All logos to new Arbimon logo (inside the app and the app icon)
- Poppins font everywhere (avoid using ABCFavorit because it might cause sizing issues)
- New color scheme matching arbimon.org

Notes:
- Avoid changes that will affect layout

### PR DESCRIPTION
**Pull request by rassokhina-e**
## ✅ DoD

- [x] Resolves #203 
- [x] Release notes updated
- [x] HelpScout documentation updates proposed  na

_(use n/a when API docs (Release notes, etc) do not need to be updated)_

## 📝 Summary

the build version 1.5.6

Untitled 3

## 📸 Examples

Untitled

Untitled1

Untitled 2

## 🛑 Problems

- Write any discovered & unresolved problems (link to created Jira issues)

## 💡 More ideas

Write any more ideas you have

### CODE REVIEW

File: src/renderer/components/LandingPage/HeaderView.vue [hunk 1]
```diff
@@ -133,12 +133,12 @@ export default {
       }
     }
     .faExternal {
-      color: $white-color;
+      color: $pitch;
```
Comment (naluinui) [inline src/renderer/components/LandingPage/HeaderView.vue:136]: Have you double checked this? Changing from white to dark is a bit risky.

File: src/renderer/components/LandingPage/HeaderView.vue [hunk 2]
```diff
@@ -133,12 +133,12 @@ export default {
...
       font-size: $default-font-size;
     }
 
     .faRefresh {
-      color: $white-color;
+      color: $pitch;
```
Comment (naluinui) [inline src/renderer/components/LandingPage/HeaderView.vue:141]: Have you double checked this? Changing from `white` to `dark` is a bit risky.

File: src/renderer/globals.scss [hunk 3]
```diff
@@ -190,3 +234,33 @@ a.is-button {
 .modal-card {
   z-index: 200;
 }
+
+.button .is-rounded .is-primary {
+  color: $pitch !important;
```
Comment (naluinui) [inline src/renderer/globals.scss:239]: are we sure to use `dark` color for button?

File: src/renderer/globals.scss [hunk 4]
```diff
@@ -190,3 +234,33 @@ a.is-button {
...
+  font-weight: 500 !important;
+}
+
+.button.is-primary {
+  color: $pitch !important;
```
Comment (naluinui) [inline src/renderer/globals.scss:244]: I think the primary color should have `frequency` background, no?

File: src/renderer/globals.scss [hunk 5]
```diff
@@ -190,3 +234,33 @@ a.is-button {
...
+  font-weight: 500 !important;
+}
+
+.button.is-primary[disabled] {
+  color: $pitch !important;
+  font-weight: 500 !important;
+}
+
+.button.is-rounded.is-cancel {
+  color: $brand-primary !important;
```
Comment (naluinui) [inline src/renderer/globals.scss:254]: Where do we use this button? It seems a bit weird to use the green color for the action that sounds risky like cancel

File: src/renderer/globals.scss [hunk 6]
```diff
@@ -190,3 +234,33 @@ a.is-button {
...
+  background-color: transparent !important;
+  font-weight: 500 !important;
+  border-color: $brand-primary !important;
+}
+
+.button {
```
Comment (naluinui) [inline src/renderer/globals.scss:260]: Can we move this to be the first of all the button configuration?

```
.button { ... }
.button .xx { ... }
.button .xx .yy { ... }
```

**Review by naluinui — CHANGES_REQUESTED** [review]

**Comment by naluinui** [comment]
Should we also try to build the windows version?

File: src/renderer/components/LandingPage/HeaderView.vue (hunk 1 above)
Comment (rassokhina-e) [reply src/renderer/components/LandingPage/HeaderView.vue:136]: @naluinui do you agree that the white color for this link will be not really readable?
Untitled
Untitled2

File: src/renderer/components/LandingPage/HeaderView.vue (hunk 2 above)
Comment (rassokhina-e) [reply src/renderer/components/LandingPage/HeaderView.vue:141]: good catch!

File: src/renderer/globals.scss (hunk 5 above)
Comment (rassokhina-e) [reply src/renderer/globals.scss:254]: this is the text color, not the background
Untitled

Comment (rassokhina-e) [reply src/renderer/globals.scss:254]: Untitled

File: src/renderer/globals.scss (hunk 4 above)
Comment (rassokhina-e) [reply src/renderer/globals.scss:244]: this is the color of the text in frequency background
Untitled

File: src/renderer/globals.scss (hunk 3 above)
Comment (rassokhina-e) [reply src/renderer/globals.scss:239]: this is the color of the text in frequency background
Untitled

File: src/renderer/globals.scss (hunk 6 above)
Comment (rassokhina-e) [reply src/renderer/globals.scss:260]: fixed

**Pull request merged**